src/autoscaler/integration/integration_suite_test.go

```diff
@@ -75,19 +75,19 @@ var _ = AfterSuite(func() {
 })
 
 var _ = BeforeEach(func() {
+	httpClient = &http.Client{}
```

Comment:
If you are going to create a client, please set the appropriate timeouts. You will need to lift the DefaultTransport definition and just modify/add timeouts as needed.